buildtools/check-no-goog.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/sh
-
-# Any call to the goog library except the followings are forbidden
-echo "Checking use of goog library..."
-if grep --include '*.js' -Rn "goog\." src contribs examples | grep -E -v 'goog.provide|goog.require|goog.module|goog.asserts|goog.exportSymbol|goog.exportProperty'
-then
-  echo "Found forbidden uses."
-  return 1
-fi
```

Comment:
FWIW, the script may still be used by projects depending on ngeo.

Reply:
You relay plan to use it on ngeo 2.4?